build/ci/cloudbuild.goldendiff.yaml

```diff
@@ -40,4 +40,4 @@ options:
   volumes:
     - name: tmp
       path: /tmp
-  machineType: "E2_HIGHCPU_32"
+  machineType: "N1_HIGHCPU_32"
```

Comment:
Could you document these build changes in the PR description?

Reply:
Done